src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#">
+        <img
+          src="images/logo.png"
+          alt="Logo Moyo"
+          width="40px"
+          height="40px"
+          class="header_logo"
+        />
+      </a>
+      <nav class="header_nav">
+        <ul class="header_nav_list">
+          <li class="header_nav_link">
+            <a
+              class="link is-active"
+              href="#"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="header_nav_link">
+            <a
+              class="link"
+              href="#"
```

Comment:
The attributes of the <a> tag should start on a new line with 2-space indentation related to the tag to keep the attributes correctly formatted.